modules/apps/asset/asset-publisher-web/src/main/java/com/liferay/asset/publisher/web/internal/util/AssetRendererUtil.java

```diff
@@ -31,11 +32,19 @@ public static String getAssetRendererUserFullName(
 			AssetRenderer assetRenderer, HttpServletRequest httpServletRequest)
 		throws PortalException {
 
-		if (assetRenderer.getUserId() > 0) {
-			User assetRendererUser = UserLocalServiceUtil.getUser(
-				assetRenderer.getUserId());
+		long assetRendererUserId = assetRenderer.getUserId();
 
-			return assetRendererUser.getFullName();
+		if (assetRendererUserId > 0) {
+			User user = null;
+
+			try {
+				user = UserLocalServiceUtil.getUser(assetRendererUserId);
```

Comment:
Hi guys, I originally discussed this issue with Allen Ziegenfus. Your solution is correct but you could simplify it a little bit by using `fetchUser` instead of `getUser`. Fetch's don't throw exceptions and return `null` if the entity does not exist.

Reply:
Thanks for the suggestion! See: https://github.com/ChrisKian/liferay-portal/pull/209/commits/a65f2b3126431fdc42e9804502035798509c0279